### ISSUE TITLE
# of classes defined in config

### ISSUE THREAD

**Issue by kaishijeng** [thuyngch/ATSS-EfficientDet-PyTorch#4]
My own dataset has 10 classes. In config, it has 81 classes for coco.
Should I replace 81 with 10 or 11 for my down dataset?

Thanks,

**Comment by thuyngch** [comment]
You should replace 11

**Comment by kaishijeng** [comment]
Thanks for the response. One more question, is it possible to do a finetuning based on your pretrained coco model on my own dataset which has only 10 classes?

Thanks,

**Comment by thuyngch** [comment]
yes absolutely ;)

**Comment by kaishijeng** [comment]
Great! Any instruction to do fine-tuning?

**Comment by thuyngch** [comment]
Please refer to https://github.com/thuyngch/ATSS-EfficientDet-PyTorch/issues/2

**Comment by kaishijeng** [comment]
I don't see anything related to fine-tuning  in #2 
For clarity, I want to load your trained d0 weight and fine-tuning with own dataset. I don't see config or train.py which allows to load your trained d0 weight
By the way my dataset is already in coco format and training from the scratch is working.
With fine-tuning approach, I am hoping I don't need to train so many epochs

**Comment by kaishijeng** [comment]
@thuyngch 

My training is stuck with range of  loss_cls: 0.1561, loss_bbox: 0.4664, loss_centerness: 0.6163, loss: 1.2387 after 27th epoch. It seems not decreasing for last several epochs. Any guidance?

2020-04-20 14:50:47,952 - mmdet - INFO - Epoch [27][3400/4367]  lr: 0.00981, eta: 10 days, 6:35:37, time: 0.727, data_time: 0.010, memory: 8366, loss_cls: 0.1561, loss_bbox: 0.4664, loss_centerness: 0.6163, loss: 1.2387
2020-04-20 14:51:24,359 - mmdet - INFO - Epoch [27][3450/4367]  lr: 0.00981, eta: 10 days, 6:34:52, time: 0.728, data_time: 0.010, memory: 8366, loss_cls: 0.1500, loss_bbox: 0.4615, loss_centerness: 0.6171, loss: 1.2286
2020-04-20 14:52:00,760 - mmdet - INFO - Epoch [27][3500/4367]  lr: 0.00981, eta: 10 days, 6:34:06, time: 0.728, data_time: 0.010, memory: 8366, loss_cls: 0.1551, loss_bbox: 0.4637, loss_centerness: 0.6168, loss: 1.2356
2020-04-20 14:52:37,109 - mmdet - INFO - Epoch [27][3550/4367]  lr: 0.00981, eta: 10 days, 6:33:20, time: 0.727, data_time: 0.010, memory: 8366, loss_cls: 0.1474, loss_bbox: 0.4625, loss_centerness: 0.6173, loss: 1.2272
2020-04-20 14:53:13,563 - mmdet - INFO - Epoch [27][3600/4367]  lr: 0.00981, eta: 10 days, 6:32:36, time: 0.729, data_time: 0.010, memory: 8366, loss_cls: 0.1505, loss_bbox: 0.4635, loss_centerness: 0.6173, loss: 1.2313
2020-04-20 14:53:49,995 - mmdet - INFO - Epoch [27][3650/4367]  lr: 0.00981, eta: 10 days, 6:31:51, time: 0.729, data_time: 0.010, memory: 8366, loss_cls: 0.1530, loss_bbox: 0.4605, loss_centerness: 0.6163, loss: 1.2299
2020-04-20 14:54:26,523 - mmdet - INFO - Epoch [27][3700/4367]  lr: 0.00981, eta: 10 days, 6:31:07, time: 0.731, data_time: 0.011, memory: 8366, loss_cls: 0.1526, loss_bbox: 0.4587, loss_centerness: 0.6155, loss: 1.2268
2020-04-20 14:55:02,966 - mmdet - INFO - Epoch [27][3750/4367]  lr: 0.00981, eta: 10 days, 6:30:22, time: 0.729, data_time: 0.010, memory: 8366, loss_cls: 0.1536, loss_bbox: 0.4684, loss_centerness: 0.6176, loss: 1.2397
2020-04-20 14:55:39,395 - mmdet - INFO - Epoch [27][3800/4367]  lr: 0.00981, eta: 10 days, 6:29:37, time: 0.729, data_time: 0.010, memory: 8366, loss_cls: 0.1533, loss_bbox: 0.4615, loss_centerness: 0.6168, loss: 1.2317

**Comment by thuyngch** [comment]
Hi @kaishijeng ,

I have already added some tips for training a custom dataset.
Please see here and feedback to me if you still need help: https://github.com/thuyngch/ATSS-EfficientDet-PyTorch#tips-for-training-on-a-custom-dataset
